app/src/main/java/bubbleheads/buy_cook/Category.java

```diff
@@ -0,0 +1,16 @@
+package bubbleheads.buy_cook;
+
+public enum Category {
```

Comment:
actually I don't like the idea to keep Categories as enum. It would mean that each time you want to add new category to app - you'll need to make new release. What if you'll want to add custom category for user. I would keep Category as class. And in future would have some method to get Category list from server